e2e/playwright.config.js

```diff
@@ -0,0 +1,58 @@
+import { defineConfig, devices } from "@playwright/test";
+
+// Load environment variables from .env file if it exists
+// import * as dotenv from 'dotenv';
+// dotenv.config();
+
+/**
+ * See https://playwright.dev/docs/test-configuration.
+ */
+export default defineConfig({
+  timeout: 20000,
+  testDir: "./tests", // Ensure this points to the correct test directory
+  /* Run tests in files in parallel */
+  fullyParallel: true,
+  /* Fail the build on CI if you accidentally left test.only in the source code. */
+  forbidOnly: !!process.env.CI,
+  /* Retry on CI only */
+  retries: process.env.CI ? 2 : 0,
+  /* Opt out of parallel tests on CI. */
+  workers: process.env.CI ? 1 : undefined,
+  /* Reporter to use. See https://playwright.dev/docs/test-reporters */
+  reporter: "html",
+  /* Shared settings for all the projects below. See https://playwright.dev/docs/api/class-testoptions. */
+  use: {
+    /* Base URL to use in actions like `await page.goto('/')`. */
+    baseURL: process.env.CI ? undefined : "http://127.0.0.1:3000",
+
+    /* Collect trace when retrying the failed test. See https://playwright.dev/docs/trace-viewer */
+    trace: "on-first-retry",
+    screenshot: "on",
+    video: "on-first-retry",
+  },
+
+  /* Configure projects for major browsers */
+  projects: [
```

Comment:
question for discussion: 🤔 I'm a little skeptical that it's valuable to have the infra template's default configuration include so many browsers. What issues would this catch? I'm biased towards simplifying this to just one project configuration.

Reply:
If this is just a base config I would propose just desktop chrome and mobile chrome as a default - and then the app specific playwright config can override with additional?

No expert here, but I noticed when developing the test spec against different browsers, Safari seemed to have some strange behavior I wasn't seeing in other browsers.

Reply:
That works for me.